wp-post-meta-revisions.php

```diff
@@ -211,4 +211,4 @@ public function _wp_preview_meta_filter( $value, $object_id, $meta_key, $single
 	}
 }
 
-$wp_post_meta_revisioning = new WP_Post_Meta_Revisioning;
+$wp_post_meta_revisioning = new WP_Post_Meta_Revisioning();
```

Comment:
how does that even work without the parenthesis?

Reply:
That just means you're implicitly not passing parameters to the constructor. Works just fine.